authority/authority.go

```diff
@@ -218,6 +220,9 @@ func (a *Authority) init() error {
 			// Append public key to list of host certs
 			a.sshCAHostCerts = append(a.sshCAHostCerts, a.sshCAHostCertSignKey.PublicKey())
 			a.sshCAHostFederatedCerts = append(a.sshCAHostFederatedCerts, a.sshCAHostCertSignKey.PublicKey())
+			// Configure template variables
+			tmplVars.SSH.HostKey = a.sshCAHostCertSignKey.PublicKey()
+			tmplVars.SSH.HostFederatedKeys = append(tmplVars.SSH.HostFederatedKeys, a.sshCAHostFederatedCerts[1:]...)
```

Comment:
Can you leave a comment here, and on the identical code below explaining what's going on here. Why are we only using [1:], etc.

Reply:
I see that you're just moving the code around, but a bit of explanation here would still help (me).